clients/wallet/src/error.rs

```diff
@@ -8,8 +8,12 @@ use thiserror::Error;
 pub enum Error {
 	#[error("Invalid secret key")]
 	InvalidSecretKey,
-	#[error("Error fetching horizon data: {0}")]
-	HorizonResponseError(#[from] ReqwestError),
+	#[error("Error fetching horizon data: reqwest: {reqwest:?}, other: {other:?}")]
+	HorizonResponseError {
+		reqwest: Option<ReqwestError>,
```

Comment:
Maybe we should rename `reqwest` to `error` because just `reqwest` somehow suggests to me that it's the client or a response and not an error. 

```suggestion
		error: Option<ReqwestError>,
```

Reply:
renamed to `error`